recipes/packages.rb

```diff
@@ -2,101 +2,8 @@
 # Cookbook Name:: windev
 # Recipe:: packages
 #
-# Copyright (c) 2014-2016 Zühlke, All Rights Reserved.
+# Copyright (c) 2014-2018 Zühlke, All Rights Reserved.
 
-include_recipe 'windev::depot'
```

Comment:
I like that the different package types are now separated in different files. 
Makes the code more readable and findable